datumaro/plugins/yolo_detection_format/format.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from enum import Enum, auto
+from typing import Dict
+import re
+
+
+class YoloDetectionPath:
+    DEFAULT_SUBSET_NAME = "train"
+    MUST_SUBSET_NAMES = ["train"]
+    ALLOWED_SUBSET_NAMES = ["train", "valid", "test", "val"]
+    RESERVED_CONFIG_KEYS = ["backup", "classes", "names"]
+    META_FILE = "data.yaml"
+
+    @staticmethod
+    def _parse_config(path: str) -> Dict[str, str]:
```

Comment:
Why did you start the method with underscore?

Reply:
This function is also used only internally in the scope of this plugin folder. I also took inspiration from the existing implementation of `yolo_format` in the repo `datumaro/plugins/yolo_format`.  Should I remove it ?